frontend/src/features/settings/account/ExportAllDataForm.tsx

```diff
@@ -44,7 +44,7 @@ const ExportAllDataForm = () => {
     const response = await usersMeExportsAllRetrieveBlob();
 
     if (response.ok === true) {
-      const zipFileBlob = response.blob();
+      const zipFileBlob = await response.blob();
```

Comment:
Fixes a Typescript error (not sure why it was not triggered by the previous version).